src/manifest.json

```diff
@@ -316,6 +318,7 @@
           "type": "internal-link",
           "id": "sq-range",
           "label": "Range queries",
+         "summary": "Query on numeric fields that returns the values that are in between a given start and end value, such as prices, dates.",
```

Comment:
```suggestion
         "summary": "Perform queries on numeric fields to return the values that are in between a given start and end value, such as prices and dates.",
```

Reply:
need to cut 10 characters to fit the screen
what do you think about:
`Perform queries on numeric fields to return the values that are between a start and end value, such as prices and dates.`